src/components/ui/file-upload.tsx

```
@@ -0,0 +1,252 @@
+            type="button"
+            onClick={handleOpenFilePicker}
+            className="flex w-full flex-col items-center gap-2 text-center"
+          >
+            <div className="flex h-10 w-10 items-center justify-center rounded-full bg-primary-5 text-primary">
+              <svg
+                width="20"
+                height="20"
+                viewBox="0 0 20 20"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+              >
```

Biome (2.3.13)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)